Rewrite 11.2-angular process for replacing encrypted disk. (#1497)
freenas/freenas-docs: userguide/storage.rst

If the ZFS pool is encrypted, additional steps are needed when
replacing a failed drive.

Ensure a :ref:`passphrase has been set <Encryption Options>`
**before** attempting to replace the failed drive. To replace a drive

> in an encrypted pool

Reviewer: ```suggestion
in an encrypted pool,
```
Feels like a comma, colon, or other character is needed here.

#. Go to 

>    :menuselection:`Storage --> Pools -->` |ui-settings| :menuselection:`--> Status`.

Reviewer: Would it be useful to insert an extra instruction after `Pools`?

``--> Pools. Locate the encrypted pool entry and click |ui-settings| :menuselection:`--> Status`.``

   Find the disk to replace and click
   |ui-options| :menuselection:`--> Replace`.
   Enter the passphrase that was set for the pool, then click
   :guilabel:`REPLACE DISK`.

#. Immediately :ref:`rekey the pool <Managing Encrypted Pools>` by
   clicking
   |pool-lock| :menuselection:`--> Encryption Rekey`.
   **If the pool is not rekeyed before the next reboot, access to
   the pool will be permanently lost**.

#.  Click
    |pool-lock| :menuselection:`--> Add Recovery Key`
    and enter the root password to save the new recovery key. The old
    recovery key will no longer function and can be safely discarded.

> .. warning:: The passphrase for a pool is reset when the pool is

Reviewer: ```suggestion
.. warning:: The pool passphrase is removed when the pool is
```
